arangod/RestHandler/RestDocumentHandler.h

```diff
@@ -35,7 +35,16 @@ class RestDocumentHandler : public RestVocbaseBaseHandler {
  public:
   RestStatus execute() override final;
   char const* name() const override final { return "RestDocumentHandler"; }
-  RequestLane lane() const override final { return RequestLane::CLIENT_SLOW; }
+  RequestLane lane() const override final {
+    bool isSyncReplication = false;
+    // We do not care for the real value, enough if it is there.
+    _request->value(StaticStrings::IsSynchronousReplicationString, isSyncReplication);
```

Comment:
You could emphasize the comment:
```suggestion
    std::ignore = _request->value(StaticStrings::IsSynchronousReplicationString, isSyncReplication);
```